digiroad2-oracle/src/test/scala/fi/liikennevirasto/digiroad2/util/DataFixture.scala

```diff
@@ -396,6 +398,71 @@ object DataFixture {
     }
   }
 
+  def verifyRoadLinkAdministrativeClassChanged(): Unit = {
+    println("\nVerify if roadlink administrator class and floating reason of mass transit stop asset was modified")
+    println(DateTime.now())
+
+    val municipalities: Seq[Int] =
+      OracleDatabase.withDynSession {
+        Queries.getMunicipalities
+      }
+
+    val administrationClassPublicId = "linkin_hallinnollinen_luokka"
+
+    OracleDatabase.withDynTransaction {
+
+      val administrationClassPropertyId = dataImporter.getPropertyTypeByPublicId(administrationClassPublicId)
+
+      municipalities.foreach { municipality =>
+        println("Start processing municipality %d".format(municipality))
+
+        println("Start verification if road link administrative class is changed")
+        verifyIsChanged(administrationClassPublicId, administrationClassPropertyId, municipality)
+
+        println("End processing municipality %d".format(municipality))
+      }
+    }
+
+    println("\n")
+    println("Complete at time: ")
+    println(DateTime.now())
+    println("\n")
+  }
+
+  private def verifyIsChanged(propertyPublicId: String, propertyId: Long, municipality: Int): Unit = {
+    case class AssetBeforeUpdate(asset: PersistedPointAsset, persistedFloating: Boolean, floatingReason: Option[FloatingReason])
+
+    val floatingReasonPublicId = "kellumisen_syy"
+    val floatingReasonPropertyId = dataImporter.getPropertyTypeByPublicId(floatingReasonPublicId)
+
+    val typeId = 10
+    //Get all no floating mass transit stops by municipality id
+    val assets = dataImporter.getNonFloatingAssetsWithNumberPropertyValue(typeId, propertyPublicId, municipality)
+
+    println("Processing %d assets not floating".format(assets.length))
+
+    if (assets.length > 0) {
+
+      val roadLinks = vvhClient.fetchVVHRoadlinks(assets.map(_._2).toSet)
+
+      assets.foreach {
+        _ match {
+          case (assetId, linkId, None) =>
+            println("Don't exist Administration Class value at the asset with id %d .".format(assetId))
```

Comment:
"Mass transit stop with asset-id: %d doesn't have Administration Class value" might be easier to understand